build/to-stylesheet.js

```diff
@@ -14,7 +14,7 @@ export const buildPropsStylesheet = ({filename,props}, {selector,prefix}) => {
     if (prop.includes('-@'))
       return
 
-    if (prefix)
+    if (prefix && prefix !== "''")
```

Comment:
```suggestion
    if (prefix && prefix !== '')
```

![Screen Shot 2022-02-23 at 9 20 22 PM](https://user-images.githubusercontent.com/1134620/155462856-ca89ea8f-bfcd-4f71-926b-5025a458be85.png)

curious why you are checking for quotes instead of empty string?

Reply:
Because that's what seems to be passed in as arg from the terminal.

While Unix systems pass in undefined for empty args for some reason Windows passes in '' and that then is interpreted as string so you get an string with quotes as an result.

Tried everything else that's the only solution that works for me. I really need to check for that.

Maybe wait until you get your windows machine back before merging and check yourself. I am on Windows 10 Node Version 16.14